src/colorizer/utils/url_utils.ts

```diff
@@ -14,6 +14,12 @@ const URL_PARAM_RANGE = "range";
 const URL_PARAM_COLOR_RAMP = "color";
 const URL_COLOR_RAMP_REVERSED_SUFFIX = "!";
 
+const ALLEN_FILE_PREFIX = "/allen/";
```

Comment:
it seems like you don't really need this prefix if you just have the remapping table below.

Reply:
I'm going to keep this in because I use it to warn users if they are using an `/allen` file link but it's not one of the ones we support. (see LoadDatasetButton.tsx, line 130)

Reply:
I'm going to keep this in because I use it to warn users if they are using an `/allen` file link but it's not one of the ones we support. (see LoadDatasetButton.tsx, line 130)